src/jenkins/BuildManifest.groovy

```diff
@@ -49,6 +49,16 @@ class BuildManifest implements Serializable {
             this.getArtifactRoot(jobName, buildNumber)
         ].join('/')
     }
+
+    public String getPackageName(String productName) {
+        String packagePrefix = [
+            productName,
+            this.build.version,
+            this.build.platform,
+            this.build.architecture,       
+        ].join('/')
+        return packagePrefix + '.tar.gz'
+    }
```

Comment:
Can we use the name with version, platform, architecture and distribution from [here](https://github.com/opensearch-project/opensearch-build/blob/ce1547934041707af62ff54dd5a21d1c8ddaf2fc/src/jenkins/InputManifest.groovy#L52-L56)?